Jantimizer/QueryOptimiser/QueryGenerators/PostgresGenerator.cs

```diff
@@ -0,0 +1,62 @@
+﻿using QueryParser.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QueryOptimiser.QueryGenerators
+{
+    public class PostgresGenerator : IQueryGenerator
+    {
+        static string Prefix = "SELECT * FROM ";
+        static string Suffix = ";";
```

Comment:
We most likely also need some additional statements to force Postgres to use the order that we give it. (Something like OPTION in MSSQL as an example)